src/enact/__init__.py

```diff
@@ -14,15 +14,22 @@
 
 """Top-level definitions."""
 
-from enact._version import __version__
+# This must be imported first, as it sets up necessary global state.
+import enact.register_enact_distribution
```

Comment:
Yeah, I think we can just add an `ensure` call to the registration logic (ensure enact distro is registered). Will get on that in mo'.

Reply:
Well, what I originally planned doesn't work due to cyclical imports, but I changed it to make it an explicit call in __init__.py rather than an import side effect. Let me know if the comment makes sense.

Reply:
Oh ok, I was wondering whether you could call enact.register_enact_distribution.ensure_registered() inside the methods that perform registration that require Enact to be registered first instead. Since this is something that is required to move the library, to avoid cyclic imports you could try merging `enact.register_enact_distribution` into `enact.distribution_registry.register_distribution`.

Reply:
Ah, makes sense. Done.